cob_phidgets/ros/include/cob_phidgets/phidgetik_ros.h

```diff
@@ -86,9 +86,9 @@ class PhidgetIKROS: public PhidgetIK
 	ros::ServiceServer _srvDigitalOut;
 	ros::ServiceServer _srvDataRate;
 	ros::ServiceServer _srvTriggerValue;
+	ros::ServiceServer _srvDigitalIn;
 
 	int _serial_num;
-	std::string _board_name;
```

Comment:
Please do not remove "_board_name". The boards serial num should only be used as uri path identifierer if no yaml configuration is present. See phidget_sensors_node.cpp line 138 to 145.